gh-pages-source/src/app/app.component.html

```diff
@@ -43,15 +43,15 @@ <h2>Getting Project Hatchway</h2>
                 <h5>vSphere Storage for Docker</h5>
                 <p>The Docker Certified Volume Driver abstracts the underlying enterprise-class storage of the vSphere environment and makes it available as Docker volumes. It is easy to install and use from a developer perspective yet leaves storage visibility and control in the hands of a vSphere administrator. It supports data services such as cloning and snapshots, ensuring that data is both resilient and highly available. Additionally, when using vSAN as the underlying storage layer, storage based policies are fully supported to take full advantage of using hyper-converged infrastructure with containers on vSphere.</p>
                 <ul>
-                  <li>Project Repository: <a href="https://github.com/vmware/docker-volume-vsphere">https://github.com/vmware/docker-volume-vsphere</a></li>
-                  <li>Documentation: <a href="https://vmware.github.io/docker-volume-vsphere/">vSphere Storage for Docker</a></li>
+                  <li>Project Repository: <a href="https://github.com/vmware/vsphere-storage-for-docker">https://github.com/vmware/vsphere-storage-for-docker</a></li>
+                  <li>Documentation: <a href="https://vmware.github.io/vsphere-storage-for-docker">vSphere Storage for Docker</a></li>
```

Comment:
Please revert this to old name, we need to take care this change after/with merging https://github.com/vmware/docker-volume-vsphere/pull/2039